frontend/src/components/worksheets/NewUpload/NewUpload.jsx

```diff
@@ -184,9 +182,7 @@ class NewUpload extends React.Component<{
             uploading: true,
         });
         let url = `/rest/bundles?worksheet=${ worksheetUUID }`;
-        if (after_sort_key) {
-            url += `&after_sort_key=${ after_sort_key }`;
-        }
+        url = this._setAfterSortKey(after_sort_key,url);

```

Comment:
Did you mean to change this too? 

```suggestion
        url += `&after_sort_key=${ isNaN(after_sort_key) ? -1: after_sort_key }`;

```